packages/adapter-sveltekit/src/hooks/project-init.ts

```diff
@@ -62,8 +62,7 @@ const createPrismicIOFile = async ({
 			 * The project's Prismic repository name.
 			 */
 			export const repositoryName =
-				import.meta.env.${PRISMIC_ENVIRONMENT_ENVIRONMENT_VARIABLE_NAME} ||
-				config.repositoryName;
+				import${"."}meta${"."}env.${PRISMIC_ENVIRONMENT_ENVIRONMENT_VARIABLE_NAME} || config.repositoryName;
```

Comment:
⚠️ We need handle this for js right?

Same for Next?!

I guess we have another bug in prod right?